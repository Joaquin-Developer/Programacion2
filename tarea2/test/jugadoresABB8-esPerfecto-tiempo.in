crearArbolJugadores
esPerfectoArbolJugadoresTiempo 10000 2
liberarArbolJugadores

crearArbolJugadores
esPerfectoArbolJugadoresTiempo2 10000 2
liberarArbolJugadores

crearArbolJugadores
esPerfectoArbolJugadoresTiempo2 16383 2
liberarArbolJugadores

Fin