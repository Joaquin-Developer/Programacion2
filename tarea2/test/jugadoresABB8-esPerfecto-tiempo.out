1>2>El árbol de jugadores NO es perfecto.
Bien.
3>4>5>El árbol de jugadores NO es perfecto.
Bien.
6>7>8>El árbol de jugadores es perfecto.
Bien.
9>10>Fin.
